samples/crypto/kmu_usage_nrf54l/boards/nrf54l15dk_nrf54l15_cpuapp_ns.conf

```diff
@@ -1,9 +1,4 @@
-#
 # Copyright (c) 2024 Nordic Semiconductor ASA
-#
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
-#
 
-# MbedTLS heap configuration
-CONFIG_MBEDTLS_ENABLE_HEAP=y
-CONFIG_MBEDTLS_HEAP_SIZE=8192
+CONFIG_TFM_ITS_ENCRYPTED=y
```

Comment:
Why is encrypted ITS required in this sample? Also ITS_ENCRYPTED is default on for 54l15